Assets/PlayerAssets/Scripts/Experience.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[System.Serializable]
+public class Experience : MonoBehaviour {
+
+    public double currentExperience;                            //how much experience the character currently has
+    public double toNextLevel;                                  //experience required to level up
+    public int charLevel;                                       //current character level
+
+	void Start(){
```

Comment:
Well I looked for a linter, but apparently c# developers are too good to have their code tell them it's formatted wrong so I'm going to be spamming you with changes so our code looks the same.

Space between Start() and {